src/aquarium-optimized/Aquarium.cpp

```diff
@@ -115,32 +115,33 @@ Aquarium::~Aquarium() {
   delete mFactory;
 }
 
-BACKENDTYPE Aquarium::getBackendType(const std::string &backendPath) {
+std::pair<BACKENDTYPE, BACKENDTYPE> Aquarium::getBackendType(
+    const std::string &backendPath) {
   if (backendPath == "opengl") {
-    return BACKENDTYPE::BACKENDTYPEOPENGL;
+    return {BACKENDTYPE::BACKENDTYPEOPENGL, BACKENDTYPE::BACKENDTYPELAST};
```

Comment:
Maybe call this {BACKENDTYPE::DIRECTCALL, BACKENDTYPE::OPENGL}? 
Same as {BACKENDTYPE::DIRECTCALL, BACKENDTYPE::D3D12}

Reply:
[1]